src/components/DirectionsButton/DirectionsButton.test.js

```diff
@@ -0,0 +1,25 @@
+import 'react-native';
```

Comment:
I was under the impression you needed this for all tests, but after taking it out locally, it works fine without it. I'll update the code to remove these `import 'react-native'` lines